pkg/docker/docker.go

```diff
@@ -347,16 +347,26 @@ func (d *stiDocker) GetOnBuild(name string) ([]string, error) {
 // and returns the image metadata
 func (d *stiDocker) CheckAndPullImage(name string) (*docker.Image, error) {
 	name = getImageName(name)
+	displayName := name
+
+	if glog.Is(3) == false {
```

Comment:
@csrwng Thank you for explanation and a link. I was out of context but now it's clear.